packages/atomic/playwright-utils/base-fixture.ts

```diff
@@ -15,7 +15,7 @@ export const makeAxeBuilder: Parameters<
   const makeAxeBuilder = () =>
     new AxeBuilder({page})
       .withTags(['wcag2a', 'wcag2aa', 'wcag21a', 'wcag21aa'])
-      .include('#code-root');
+      .include('#storybook-root');
```

Comment:
We scan the whole story now: it allows to catch some sneaky issues, and to have better samples/stories.